eth/filters/bor_filter_test.go

```diff
@@ -0,0 +1,164 @@
+package filters
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/rawdb"
+	types "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/params"
+
+	gomock "github.com/golang/mock/gomock"
+)
+
+var (
+	key1, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
+	addr    = crypto.PubkeyToAddress(key1.PublicKey)
+
+	hash1 = common.BytesToHash([]byte("topic1"))
```

Comment:
could we remove package-level variables? I believe we only need a bunch of hashes that can be made inside a test or in a function.